content/en/blog/news/2022-02-15-survey-results.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
why 2 blog posts?

Reply:
I wanted to have one post with summary and overview, and a page with more detailed info. The detail page should only appear as a link in the overview and not as a separate blog with an entry in the menu. But i couldn't figure out how to do that.

Reply:
You can try `toc_hide: true` and `hide_summary: true` in the markdown front matter. That should atleast hide it from the sidebar. See: https://www.docsy.dev/docs/adding-content/navigation/#section-menu